xbitinfo/xbitinfo.py

```diff
@@ -184,6 +185,8 @@ def get_bitinformation(  # noqa: C901
     """
     if implementation == "julia" and not julia_installed:
         raise ImportError('Please install julia or use implementation="python".')
+    if ds.is_null().any():
```

Comment:
```suggestion
    if ds.isnull().any():
```
My bad, it is `isnull` and not `is_null`